week2/homework/src/index.js

```diff
@@ -1,3 +1,114 @@
 'use strict';
 
-// TODO: Write the homework code in this file
+const {
+  prompt,
+  addPrompt,
+  removePrompt,
+  completePrompt,
+  resetPrompt,
+  readPrompt,
+  updatePrompt,
+  updateQuestion,
+  exportPrompt,
+  searchPrompt
+} = require('./prompt');
+
+const chalk = require('chalk');
+const yargs = require('yargs');
+const listToDos = require('./listToDos');
+const addToDo = require('./addToDo');
+const removeToDo = require('./removeToDo');
+const completeToDo = require('./completeToDo');
+const resetToDos = require('./resetToDos');
+const readToDo = require('./readToDo');
+const updateToDo = require('./updateToDo');
+const exportToDos = require('./exportToDos');
+const searchToDos = require('./searchToDos');
+
+yargs
+  .command({
+    command: 'list',
+    describe: 'List to-dos',
+    handler: function() {
+      listToDos();
+    }
+  })
+  .command({
+    command: 'add',
+    describe: 'Add a new to-do',
+    builder: {
+      complete: {
+        describe: 'To-do complete',
+        type: 'string',
+        default: '➖'
+      }
+    },
+    handler: async function(argv) {
+      const answer = await prompt(addPrompt);
+      answer.length >= 5 ? addToDo(answer, argv.complete) : this.handler(argv);
```

Comment:
This complete feature isn't working properly unless the user have ✔emoji, which isn't always possible on all platforms.

I would change line 43 to be an empty string, and once the user inserting something as `--complete=true` it will be affected to be the emoji you want ✔. :)